src/main/webapp/util.js

```diff
@@ -103,15 +103,15 @@ function xhrDelete(url, callback, errback){
 }
 
 function parseJson(str){
-	return JSON.parse(str);
+	return window.JSON ? JSON.parse(str) : eval('(' + str + ')');
 }
 
 function objectToQuery(map){
 	var enc = encodeURIComponent, pairs = [];
 	for(var name in map){
 		var value = map[name];
 		var assign = enc(name) + "=";
-		if(value && value instanceof Array){
+		if(value && (value instanceof Array || typeof value == 'array')){
```

Comment:
```
(function() {
var arr = [];
alert(typeof arr == "array");
})();
```

Output of this function is false, therefore the typeof check here will always be false.